Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
@GrahamCampbell this is the "expected" part, input is here: https://github.com/SpacePossum/PHP-CS-Fixer/blob/master_1421_method_separation_tweaks/Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php#L382-L385

So you want that the newline is added?